libs/knowledge-store/ragstack_knowledge_store/knowledge_store.py

```diff
@@ -1,47 +1,65 @@
 import secrets
-from dataclasses import dataclass
+from dataclasses import dataclass, field
+from enum import Enum
 from typing import (
-    Any,
     Iterable,
     List,
     NamedTuple,
     Optional,
     Sequence,
-    Type,
 )
 
 import numpy as np
 from cassandra.cluster import ConsistencyLevel, ResponseFuture, Session
 from cassio.config import check_resolve_keyspace, check_resolve_session
-from langchain_community.utilities.cassandra import SetupMode
-from langchain_community.utils.math import cosine_similarity
-from langchain_core.documents import Document
-from langchain_core.embeddings import Embeddings
 
 from ragstack_knowledge_store.edge_extractor import EdgeExtractor
```

Comment:
Why do we mix `ragstack_knowledge_store.edge_extractor` and `._utils`? Should we use all relative or all absolute imports?

Reply:
fixed.